src/Engine/Spells/Spells.cpp

```diff
@@ -754,7 +774,8 @@ void EventCastSpell(int uSpellID, PLAYER_SKILL_MASTERY skillMastery, PLAYER_SKIL
                 spell_expire_time, skillMastery, spell_power, 0, 0);
             //    pAudioPlayer->PlaySound((SoundID)word_4EE088_sound_ids[uSpellID],
             //    0, 0, fromx, fromy, 0, 0, 0);
-            pAudioPlayer->PlaySpellSound(uSpellID, 0);
+            //    PID was 0
+            pAudioPlayer->PlaySpellSound(uSpellID, PID_INVALID);
```

Comment:
Shouldn't it be `PID(OBJECT_Player, 0)` instead?

Reply:
OK, I've read a bit more through the code, using `OBJECT_Player` here would be a bad idea.

Reply:
Yeah was my first idea but for OBJECT_Player voice volume is used.